src/components/LoginSignUpButton/LoginSignUpButton.jsx

```diff
@@ -0,0 +1,11 @@
+import './LoginSignUpButton.scss';
+
+const LoginSignUpButton = ({ className, onClick, disabled, buttonText }) => {
+  return (
+    <button className={className} onClick={onClick} disabled={disabled}>
+      {buttonText}
+    </button>
+  );
+};
```

Comment:
버튼에 출력되는 문구는 기존에 버튼을 사용하는 것과 같이(e.g. `<button>텍스트</button>`) 작성하는 것이, 해당 컴포넌트를 사용할 때 버튼 형태임을 명확히 해줄 것 같습니다.

props 대신 children으로 받아서 구현하는 방식으로 수정해 주세요!
```suggestion
const LoginSignUpButton = ({ className, onClick, disabled, children }) => {
  return (
    <button className={className} onClick={onClick} disabled={disabled}>
      {children}
    </button>
  );
};
```